src/components/Grid/createAddItemFooter.tsx

```
@@ -0,0 +1,62 @@
+            disabled={disabled}
+            variant="outlined"
+            color="primary"
+            size="small"
+            sx={{ minWidth: 'auto', px: 1 }}
+          >
+            <Add />
+          </Button>
+        </Tooltip>
+      </Stack>
+    );
+  };
```

ESLint

[error] Component definition is missing display name

(react/display-name)